lib/grape/validations/params_scope.rb

```diff
@@ -140,9 +140,14 @@ def require_optional_fields(context, opts)
       end
 
       def validate_attributes(attrs, opts, &block)
```

Comment:
Try to make it simpler for now, if not splitting is less code, we should go with that.